### ISSUE TITLE
unknown command "tendermint" for "osmosisd"

### ISSUE THREAD

**Issue by ljunokas** [osmosis-labs/osmosis#1350]
Hey i followed these instructions o set up a node, after it synced and wanted to set it up as validator but this comand fails:
```
user@Osmosis:~/.osmosisd/config# osmosisd tendermint show-validator
Error: unknown command "tendermint" for "osmosisd"
Run 'osmosisd --help' for usage.
```

here is available commands: 
```
user@Osmosis:~/.osmosisd/config# osmosisd --help
Start osmosis app

Usage:
  osmosisd [command]

Available Commands:
  add-genesis-account      Add a genesis account to genesis.json
  add-wasm-genesis-message Wasm genesis subcommands
  collect-gentxs           Collect genesis txs and output a genesis.json file
  config                   Create or query an application CLI configuration file
  debug                    Tool for helping with debugging your application
  export                   Export state to JSON
  export-derive-balances   Export a derive balances from a provided genesis export
  gentx                    Generate a genesis tx carrying a self delegation
  help                     Help about any command
  init                     Initialize private validator, p2p, genesis, and application configuration files
  keys                     Manage your application's keys
  migrate                  Migrate genesis to a specified target version
  prepare-genesis          Prepare a genesis file with initial setup
  query                    Querying subcommands
  rollback                 rollback tendermint state by one height
  rosetta                  spin up a rosetta server
  start                    Run the full node
  status                   Query remote node for status
  testnet                  Initialize files for a simapp testnet
  tx                       Transactions subcommands
  unsafe-reset-all         (unsafe) Remove all the data and WAL, reset this node's validator to genesis state
  validate-genesis         validates the genesis file at the default location or at the location passed as an arg
  version                  Print the application binary version information

Flags:
  -h, --help                help for osmosisd
      --home string         directory for config and data (default "/root/.osmosisd")
      --log_format string   The logging format (json|plain) (default "plain")
      --log_level string    The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
      --trace               print out full stack trace on errors

Use "osmosisd [command] --help" for more information about a command.
```

any tips on how to get validaotr pubkey?

**Comment by ValarDragon** [comment]
Ah sorry, this was broken on v7.2.0. Should be fixed in the next release, maybe we should make it soon. cc @czarcas7ic is the v7.x branch up to date with your fix on the SDK fork? If so, maybe we just release a v7.2.1?

**Comment by czarcas7ic** [comment]
@ValarDragon The v7.x branch is not yet up to date but we can do that now! @p0mvn is going to backport the pruning/refactor changed and then we will make a v0.45.0x-osmo-v7.8 sdk release for v7.x

**Comment by czarcas7ic** [comment]
Updated here https://github.com/osmosis-labs/osmosis/pull/1357

**Comment by ValarDragon** [comment]
Can you try using `v7.2.1` and see if that fixes this?

**Comment by czarcas7ic** [comment]
Closing as this is fixed in v7.2.1
